Gemfile

```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
```

Comment:
What the hell? O_o' This works???

Reply:
You mean the symlink? Yep! But it's not really necessary, we could also set the `BUNDLE_GEMFILE` env variable in our environments to point to the right `Gemfile`.

Reply:
Oh yes, I didn't realize it's a symlink and I guess the GitHub UI displays symlinks weirdly. It's OK for me if we can avoid using `dotenv` or similar, I don't like messing with other people's env :/